### ISSUE TITLE
WSL: "Can't find all parsed annotation files ... need to run parse_annotations, parse_ncbi or parse_phytozome"

### ISSUE THREAD

**Issue by DrPintoThe2nd** [jtlovell/GENESPACE#2]
Hi John,

I'm trying out this program using some datasets that I have previously run through MCScanX and I'm hitting an issue that I'm not savvy enough to work around.

Background: I've got Genespace running on WSL in its own conda environment with a fresh R install. I have both gff files that MCScanX has already accepted as "A-okay" and the raw versions from GenBank, ENSEMBL, etc for reptiles. I've tried them both and I received the same error that Google is unable to provide a meaningful suggestion: 

`Can't find all parsed annotation files ... need to run parse_annotations, parse_ncbi or parse_phytozome`

Am I missing something here? Presumably my MCScanX gffs are 'parsed' and I'm unsure where to find these scripts..

Thanks!!

P.S. With WSL, I was unable to get R to install Genespace as specified in the docs, but the fix was to call it to install the xml2 package independently and off we went:
i.e.
`if (!requireNamespace("devtools", quietly = TRUE))
	install.packages("xml2", dependencies=TRUE, INSTALL_opts = c('--no-lock'))
    install.packages("devtools")
	
if (!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")
BiocManager::install(c("Biostrings", "rtracklayer"))

devtools::install_github("jtlovell/GENESPACE", upgrade = F)

library(GENESPACE)
`

Best,
bjp

**Comment by jtlovell** [comment]
This is the expected message on the initial start up - telling you before you do anything you need to parse the annotations. But are you getting it after running `parse_annotations`, `parse_ncbi` or `parse_phytozome` to get your annotation files in order? If so, I'll look into it.

**Comment by jtlovell** [comment]
Ah - sorry, you are trying to run it on MCScanX-formatted annotations. That will not work for GENESPACE. There is too much information lost in those parsed files to do what GENESPACE needs. You need to feed it gff3 and peptide fasta formatted annotations.

**Comment by DrPintoThe2nd** [comment]
Hi John,

So I've tried running it on both MCScanX formatted and raw gff3 files to the same "error message". What I'm gathering from you so far is that (1) this is a default message, not an error message, and (2) I need to run `parse_annotations`, `parse_ncbi` or `parse_phytozome` on these gff files first.

I pulled a full stop because I thought this was an error message, not a warning/info message. I'll continue down the pipeline this week and report back.

Thanks!!
Best,
bjp
